students/msirisha/lesson5/mailroom3.py

```diff
@@ -0,0 +1,103 @@
+# Mailroom script
+# part3. send a thank you note to donors.
+# add exceptions concept.
+
+import sys
+
+#global data structure
+donor_data = { "sai emani": {"first_name" : "sai", "last_name" : "emani", "donations" : [20.23, 30.456, 50.786]},
```

Comment:
I think this dictionary is a bit overkill.  I think you would make your life a lot easier with just name, and list of donations:
```
{"sai emai": [20.23, 30.456, 50.786]}
```